.github/workflows/continuous_deployment.yml

```
@@ -0,0 +1,35 @@
+        uses: actions/checkout@v3
+
+      - name: Set up SSH
+        uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.AZURE_VM_SSH_KEY }}
+
+      - name: Deploy to Azure VM
+        run: | # Copy the docker-compose file and any other necessary files
+          scp -o StrictHostKeyChecking=no docker-compose.dev.yml ${{ secrets.AZURE_VM_USER }}@${{ secrets.AZURE_VM_IP }}:~/docker-compose.yml
+          scp -o StrictHostKeyChecking=no -r nginx ${{ secrets.AZURE_VM_USER }}@${{ secrets.AZURE_VM_IP }}:~/nginx
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)